Documentation_displayed_on_the_portal/Share_Dataset_Instruction_Page.md

```diff
@@ -0,0 +1,222 @@
+- [6) Obtention of a Digital Object Identifier (DOI) for your dataset](#doi_obtention)
+- [7) Add the new dataset to the list of CONP datasets](#addition_to_conp-dataset) (a.k.a. https://github.com/CONP-PCNO/conp-dataset)
+
+#### <a name="datalad_install"></a> 1) Installation of DataLad
+
+- **_on Linux_**
+
+We recommend the Miniconda installation procedure detailed in the [Install DataLad on linux-machines with no root access](http://handbook.datalad.org/en/latest/intro/installation.html#linux-machines-with-no-root-access-e-g-hpc-systems) entry on the [DataLad Handbook](http://handbook.datalad.org/en/latest/index.html), which installs the most up-to-date versions of DataLad, git-annex, and git if needed.
+
+- **_on Mac OS X_**
+
+Please visit the Install DataLad section of the [DataLad Handbook](http://handbook.datalad.org/en/latest/index.html) and follow the instruction for [Mac OS X install](http://handbook.datalad.org/en/latest/intro/installation.html#macos-osx).
+
+
+#### <a name="dataset_creation"></a> 2) Creation of a new DataLad dataset
+
+The first step to upload a dataset to CONP via DataLad requires the creation of a new DataLad dataset that will be tracked on GitHub.
+
+- Create a local DataLad directory:
+
+  ```
+  datalad create <new_dataset_name>
+  ```
+
+- Create a sibling for your dataset on GitHub. The command below will generate a sibling in your local space:
+
+  ```
+  cd <new_dataset_name>
+  datalad create-sibling-github <new_dataset_name>
+  ```
+  To inspect existing siblings, run `datalad siblings`
+	
+#### <a name="dataset_population"></a> 3) Population of the new dataset
+
+Choice of how to populate a new dataset will vary based on the special remote providing access to the data. The following procedure covers working with the [web](https://git-annex.branchable.com/tips/using_the_web_as_a_special_remote/) special remote. 
+
+**Dataset requirements:**
+
+- All datasets must include a `README.md` file in the root directory. That file will be displayed on the CONP portal page that will describe the dataset.
+- All datasets must include a `DATS.json` file as described in [the main documentation page](https://github.com/CONP-PCNO/conp-documentation/blob/master/CONP_DATS_fields.md). The content of this file will also be displayed on the CONP portal page that will describe the dataset. We provide a [DATS GUI editor]() ***(#TODO: ADD LINK TO EDITOR)*** for easy creation of this file. 
+- A logo can be added at the root of the dataset. This is optional, however, if there is a logo at the root of the dataset, it will be used on the CONP portal page that will describe the dataset.
+
+
+**How to track the different files of the dataset:**
+
+All files in the dataset must be added to the repository using one of the two commands below. Copying content from another location into your local copy of the repository without using those commands will not work.
```

Comment:
Maybe emphasise "will not work" in bold?  That is a very easy mistake to make, I speak from experience.

Reply:
ok